setup.sh

```diff
@@ -49,14 +49,27 @@ clean(){
 # Install project dependencies.
 install(){
   echo 'Installing project dependencies...'
+
+  # Copy globally-installed packages.
+  # Protractor - JavaScript acceptance testing.
+  if [ $(program_is_installed protractor) = 0 ]; then
+    echo 'Installing Protractor dependencies locally...'
+    npm install protractor
+    ./$NODE_DIR/protractor/bin/webdriver-manager update
+  else
+    echo 'Global Protractor installed. Copying global install locally...'
+    protractor_symlink=$(which protractor)
+    protractor_binary=$(readlink $protractor_symlink)
+    protractor_full_path=$(dirname $protractor_symlink)/$(dirname $protractor_binary)/../../protractor
+    mkdir ./$NODE_DIR/ && mkdir ./$NODE_DIR/protractor
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/d07d46e421a05c0212011242cc39fa181b540c0a